Update wrapper.py

dinabandhu50 opened the pull request:
This is regarding the issue "Prevent Padel Splash Image from Loading #14". The following code will help suppress the Padel Splash Image while running in the jupyter notebook or in python file.  headless padelpy.

tjkessler commented:
@dinabandhu50,

Nice work, thanks for helping out with issue https://github.com/ecrl/padelpy/issues/14! Just a few suggestions/questions before we merge:

1.) Let's place the new argument `headless` at the end of the function definition, just in case some people are using positional arguments instead of keyword arguments

2.) A new argument necessitates an addition to the function's docstring, let's make sure we include an explanation of the new argument

2.) Is the min/max memory allocation in the call `java -Xms1G -Xmx1G -Djava.awt.headless=true -jar` necessary? I tested the implementation without specifying memory allocation and it seems to work fine

Again, nice work, let's address the above comments and work towards merging.

-Travis

dinabandhu50 commented:
Hi @tjkessler,

Thank you for your feedbacks, and for this amazing software "a simple implementation yet very powerful and  useful"  I am using it extensively for my projects and works. And, I have done some modifications to the code, please find them in the following points
1.  Placed the headless argument in the end of the function definition.
2. Added a description about headless argument in the doc string of function definition.
3.  Changed `java -Xms1G -Xmx1G -Djava.awt.headless=true -jar` to `java -Djava.awt.headless=true -jar` as it is not necessary and not properly delivering what the argument is saying.
4.  Also added the description in the README file, in the ending of "Command Line Wrapper" section.

Again thanks for the amazing software.

- Dinabandhu

tjkessler commented:
Looks good! I'll merge, wrap up versioning, and push to PyPI.
